### PR TITLE
Add sentence about alert deactivation

### PR DESCRIPTION
**Pull request by idmyn**
To clarify what deactivating an alert actually does

### CODE REVIEW

File: docs/platform/navigation/settings/alerts.md [hunk 1]
```diff
@@ -38,7 +38,8 @@ If the owner of an alert is deleted, the associated alerts will also be deleted.
 - a destination: internal (Conduktor Console) or external (Slack, Teams or Webhook). Each alert can have **only one external destination**. To verify that it works, click **Send test**.
 
 ## Manage alerts
-You can disable an alert temporarily without deleting it. You can also test the alert at any point or duplicate it (e.g. if you want the same alert to go to different external destinations). 
+You can deactivate an alert temporarily without deleting it. The status of deactivated alerts will continue to update but will not be recorded in the alert's history, and notifications will not be fired.
+You can also test the alert at any point or duplicate it (e.g. if you want the same alert to go to different external destinations).
```
[the remark this answers is not in the document]
Comment (idmyn) [reply docs/platform/navigation/settings/alerts.md:42]: how about

```suggestion
You can also test, duplicate or delete the alert at any point.
```
